### PR TITLE
Age of mythology cheat sheet

### PR DESCRIPTION
**Pull request by Edvac**
**What does your Instant Answer do?**
Provides a cheat sheet about the game "Age of Mythology"

**What is the data source for your Instant Answer? (Provide a link if possible)**
It is a fans 's site about this game.
http://aom.heavengames.com/gameinfo/cheats

**Why did you choose this data source?**
Because it contains all the information from other sources that I found.

**Are there any other alternative (better) data sources?**
A good one could be http://www.gamefaqs.com/pc/476277-age-of-mythology/cheats

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Gamers
**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
yes->https://duck.co/ideas/idea/6387/age-of-mythology-cheatsheet

**Are you having any problems? Do you need our help with anything?**
No, thank you.

**Where did you hear about DuckDuckHack? (For first time contributors)**
I find out digging into the duckduckgo site. Link after link.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
age-of-mythology-cheatsheet

----

IA Page: https://duck.co/ia/view/age_of_mythology_cheatsheet

### CODE REVIEW

**Comment by moollaza** [comment]
@Edvac thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/json/age-of-mythology.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{
+    "id": "age_of_mythology_cheat_sheet", 
+    "name": "Codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/age-of-mythology.json]: Please change to "Age of Mythology Cheat Codes"

File: share/goodie/cheat_sheets/json/age-of-mythology.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    "description": "Aom codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/age-of-mythology.json]: Let's make this a little more descriptive, or remove it if you think it's not necessary. As long as it's nothing redundant :+1:

File: share/goodie/cheat_sheets/json/age-of-mythology.json [hunk 3]
```diff
@@ -0,0 +1,111 @@
+{
+    "id": "age_of_mythology_cheat_sheet", 
+    "name": "Codes",
+    "description": "Aom codes", 
+    "metadata": { 
+        "sourceName": "Aom codes",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/age-of-mythology.json]: Let's change this to `HeavenGames`

File: share/goodie/cheat_sheets/json/age-of-mythology.json [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        "sourceUrl": "http://aom.heavengames.com/gameinfo/cheats"
+    },
+     "aliases": [
+        "age of mythology cheats",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/age-of-mythology.json]: Just so you're aware, these aliases are not exact triggers that will show this -- a trigger word from "CheatSheets.pm" is currently required to indicate the Cheat Sheet controller needs to analyze the query. So for these aliases to work you still need "reference", "help", "cheat sheet", etc to be in the query.

**Comment by moollaza** [comment]
@Edvac thanks again for contributing, we really appreciate it! I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

File: share/goodie/cheat_sheets/json/age-of-mythology.json (hunk 4 above)
Comment (Edvac) [inline share/goodie/cheat_sheets/json/age-of-mythology.json]: Thank you for making it clear. I wasn't quite sure. Do you think that they are useless? Should I remove them? For example, Probably none will type "age of mythology cheats and codes cheat sheet". I will remove the first two aliases.

**Comment by Edvac** [comment]
Hello, 
Thanks for trying to make my cheat sheet better.
I am still new to git, and the procedure frustrates me a little bit.
Do you think it is a good idea for the community to make a video for beginners?

**Comment by moollaza** [comment]
@Edvac this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@Edvac congrats! It's live: https://duckduckgo.com/?q=age+of+mythology+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by Edvac** [comment]
I'd like to continue contributing. I was wandering... Can someone make a
post at tweeter with my cheatsheet?
I would like to see if people like this cheatsheet.

**Comment by moollaza** [comment]
@Edvac we announce all the new cheat sheets...slowly :+1:

**Comment by moollaza** [comment]
There's a bit of backlog, but everything should get mentioned eventually

**Comment by Edvac** [comment]
Ok, no problem! :-) I am always happy to help the community.

**Pull request merged**